bobweb/bob/command_users.py

```diff
@@ -21,31 +22,31 @@ def __init__(self):
         )
 
     async def handle_update(self, update: Update, context: CallbackContext = None):
-        await users_command(update)
+        content = await create_users_score_formatted_table(update.effective_chat.id)
+        await update.effective_chat.send_message(content, parse_mode=ParseMode.MARKDOWN)
 
 
-async def users_command(update: Update):
-    chat_members: List[ChatMember] = database.get_chat_members_for_chat(chat_id=update.effective_chat.id)
+async def create_users_score_formatted_table(chat_id: int) -> str:
+    chat_members: List[ChatMember] = list(database.get_chat_members_for_chat(chat_id=chat_id))
     chat_members = exclude_possible_bots(chat_members)
 
     headings = ['Nimi', 'A', 'K', 'V']
     # First make list of rows. Each row is single users data
     member_array = create_member_array(chat_members)
     member_array.insert(0, headings)
 
-    formatter = MessageArrayFormatter('⌇ ', '=').with_truncation(28, 0)
+    formatter = (MessageArrayFormatter('⌇ ', '=')
+                 .with_truncation(28, 0))
     formatted_members_array_str = formatter.format(member_array)
 
     footer = 'A=Arvo, K=Kunnia, V=Viestit'
 
     # '\U0001F913' => nerd emoji, '```' =>  markdown code block
-    reply_text = 'Käyttäjät \U0001F913\n\n' \
-                 + '```\n' \
-                 + f'{formatted_members_array_str}' \
-                 + f'```\n' \
-                 + f'{footer}'
-
-    await update.effective_chat.send_message(reply_text, parse_mode=ParseMode.MARKDOWN)
+    return 'Käyttäjät \U0001F913\n\n' \
```

Comment:
Don't know if it is feasible but would maybe be neat to have all the finnish messages (and templates) that the bot can send at a separate localization directory, and the the code would just reference there.

This way, it would be easy to proof-read and at glance you would see the kind of things that the bot can say.

Reply:
Yes, separate localization should have been done from the get go and might be quite easy with Django. Now it requires way more work as non-lozalized text is all over the place.

Reply:
Easy solution would be to just all localizations to one dict where keys are locales and values dicts of localizations. Then have locale information for each chat and add one method for fetching lozalized text. Might do that later.